googletest/cmake/internal_utils.cmake

```diff
@@ -176,13 +176,11 @@ function(cxx_library_with_type name type cxx_flags)
   endif()
   if (DEFINED GTEST_HAS_PTHREAD)
     # add pthread to this library and all librariers which link to this library if needed
-    set_property(TARGET ${name} APPEND PROPERTY LINK_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
-    set_property(TARGET ${name} APPEND PROPERTY INTERFACE_LINK_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
+    target_link_libraries(${name} PUBLIC ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Current required cmake version 2.6.4

Reply:
So it was target_link_libraries before but I added the PUBLIC option.  Is that what you are alluding to when referencing V2.6.4?  If so would preference be to use old target_link_libraries with an addition set_property or use two set_properties?

Reply:
Never mind, I thought that `target_link_libraries` was added to CMake 3.0